Library/Homebrew/formulary.rb

```diff
@@ -118,6 +121,7 @@ def initialize(alias_path)
       path = alias_path.resolved_path
       name = path.basename(".rb").to_s
       super name, path
+      @install_name = alias_path.basename.to_s
```

Comment:
I was wondering if things would 💥 if we used the `@install_name` as the actual value for `name` instead? That means we could avoid having to add new fields. Thoughts?